modules/openshift-developer-cli-installing-odo-on-linux.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/installing-odo.adoc     
+
+[id="installing-odo-on-linux"]
+
+= Installing {odo-title} on Linux
+
+== RPM installation
+* For CentOS or RHEL use:
++
+----
+$ yum install odo
+----
+* For Fedora use:
++
+----
+$ dnf install odo
+----
+
+== Binary installation
+
+----
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64 -o /usr/local/bin/odo && chmod +x /usr/local/bin/odo
```

Comment:
This command will not work unless you are root..

This should be multiple lines as well instead with `$`:

```sh
$ curl -L https://mirror.openshift.com/pub/openshift-v4/clients/odo/latest/odo-darwin-amd64 -o /usr/local/bin/odo 
$ chmod +x /usr/local/bin/odo
```

Reply:
We indicate root with '#'.

Reply:
ah ok. could we still do it multiple lines @boczkowska ?

Reply:
Done with all of the installation procedures.